cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -242,7 +243,8 @@ def _clear_ftp():
     # Check it on FTP
     if log_depot.protocol != 'dropbox':
         check_ftp(appliance=appliance, ftp=log_depot.ftp, server_name=appliance.server.name,
-                  server_zone_id=appliance.server.zone.id, check_ansible_logs=True)
+                  server_zone_id=appliance.server.sid,
```

Comment:
What do you think about putting `check_ansible_logs=True` on the same line?

Reply:
Thanks! I think it's a good idea. Moved.